statsmodels/discrete/discrete_model.py

```diff
@@ -2525,22 +2525,23 @@ def _score_nbin(self, params, Q=0):
         y = self.endog[:,None]
         mu = self.predict(params)[:,None]
         a1 = 1/alpha * mu**Q
+
+        digamma_part = (special.digamma(y + a1) -
+                        special.digamma(a1))
+
         if Q: # nb1
             dparams = exog*mu/alpha*(np.log(1/(alpha + 1)) +
-                       special.digamma(y + mu/alpha) -
-                       special.digamma(mu/alpha))
+                       digamma_part)
```

Comment:
Is Q always 1 here?

Reply:
Yes.

Reply:
I know nothing about these models, but it looks like  `NegativeBinomial` with `loglike_method == 'nb2'` would result in calls with `Q=0`?

Reply:
Oh, never mind, I see